src/pages/feedback/ui/FeedbackPage.tsx

```diff
@@ -0,0 +1,36 @@
+import cl from "./FeedbackPage.module.scss"
+import PeopleIcon from "shared/assets/img/people.svg"
+import { Navigate } from 'react-router-dom';
+import { getCompletedSurvey, getQuestions } from "app/providers/router/routeConfig/routes";
+import { useQuestionsCompleted } from "shared/lib/hooks/useQuestionsCompited/useQuestionsCompited";
+import { useAppSelector } from "shared/lib/hooks/useAppSelector/useAppSelector";
+import { useAppDispatch } from "shared/lib/hooks/useAppDispatch/useAppDispatch";
+import { useEffect } from "react";
+import { fetchQuestions } from "entities/questions";
+import { FeedbackRateQuestions } from "features/feedbackRateQuestions/ui/FeedbackRateQuestions";
+
+
+const FeedbackPage = () => {
+    const {isCompleted} = useQuestionsCompleted()
+    
+    const dispatch = useAppDispatch();
+    useEffect(() => {
+        dispatch(fetchQuestions());
+    }, [dispatch]);
+
+    return isCompleted ? (
+        <Navigate to={getCompletedSurvey()} />
+    ) : (
+        <main className={cl.FeedbackPage}>
+            <div className={cl.iconContainer} >
+                <PeopleIcon className={cl.peopleIcon}/>
+            </div>
+            <div className={cl.infoBlock}>
+                <h1 className={cl.title}>Уважаемый клиент!</h1>
+                <FeedbackRateQuestions />
+            </div>
+        </main>
+    );
+};
```

Comment:
по поводу вот этого подхода с ` isCompleted ?`
в целом так тоже можно, но мне как будто больше хочется это сделать в `useEffect`

```suggestion
const FeedbackPage = (): JSX.Element | null => {
    const {isCompleted} = useQuestionsCompleted();
    
    const navigate = useNavigate();

    const dispatch = useAppDispatch();
    
     useEffect(() => {
        if (isCompleted) {
            navigate(getCompletedSurvey()
        }
    }, [isCompleted]);
    
    useEffect(() => {
        dispatch(fetchQuestions());
    }, [dispatch]);

    if (isCompleted) return null;

    return (
        <main className={cl.FeedbackPage}>
            <div className={cl.iconContainer} >
                <PeopleIcon className={cl.peopleIcon}/>
            </div>
            <div className={cl.infoBlock}>
                <h1 className={cl.title}>Уважаемый клиент!</h1>
                <FeedbackRateQuestions />
            </div>
        </main>
    );
};
```

Reply:
аналогично в других местах
на мой взгляд, так выглядит правильнее с семантической точки зрения, так как ты вообще ничего не должен отображать, а только редиректить